orte/mca/ras/slurm/ras_slurm_component.c

```diff
@@ -106,6 +106,14 @@ static int ras_slurm_register(void)
                                             MCA_BASE_VAR_SCOPE_READONLY,
                                             &mca_ras_slurm_component.rolling_alloc);
 
+    mca_ras_slurm_component.use_all = false;
+    (void) mca_base_component_var_register (component, "use_entire_allocation",
+                                            "Use entire allocation (not just job step nodes) for this application",
+                                            MCA_BASE_VAR_TYPE_BOOL, NULL, 0, 0,
+                                            OPAL_INFO_LVL_9,
```

Comment:
I confess that I've given up on the levels, Jeff. There are just too many of them, and nobody really understands them in the user community. I'll put it to a 5 just to set it to something, but we really should address this someday.